.gitignore

```diff
@@ -1,3 +1,21 @@
-target
+# IDE ignores
+.history
 .idea
 *.iml
+.settings
+.vscode
+.classpath
+.project
+.DS_Store
+
+# Java ignores
+target
+src/main/resources/webroot/*
```

Comment:
I guess the flow is that it builds the UI into this folder, then runs the mvn build? I think we can probably get it to do this in a better way, but happy to leave it like this for now.

Reply:
Correct